backend/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nerds-against-society.backend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "ts-node-dev --respawn --pretty --transpile-only src/index.ts",
+    "build": "tsc --project ./",
+    "test": "jest",
+    "prepare": "cd .. && husky install backend/.husky"
+  },
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "helmet": "^4.5.0",
+    "socket.io": "^4.0.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.16",
+    "@babel/preset-env": "^7.13.15",
+    "@babel/preset-typescript": "^7.13.0",
+    "@types/cors": "^2.8.10",
+    "@types/dotenv": "^8.2.0",
+    "@types/express": "^4.17.11",
+    "@types/helmet": "^4.0.0",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^14.14.41",
+    "@types/socket.io": "^2.1.13",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
+    "babel-jest": "^26.6.3",
+    "eslint": "^7.25.0",
+    "eslint-config-prettier": "^8.3.0",
+    "husky": "^6.0.0",
+    "jest": "^26.6.3",
+    "lint-staged": "^10.5.4",
+    "prettier": "2.2.1",
+    "ts-node-dev": "^1.1.6",
+    "typescript": "^4.2.4"
+  },
+  "lint-staged": {
+    "**/*": "prettier --write --ignore-unknown"
+  }
```

Comment:
Just to clarify - this only runs on staged files and is specifically for husky?

Reply:
Yep so Husky itself is the Git hook & runs as a 'pre-commit' each time `git commit` is invoked. It calls `lint-staged` in turn which pipes only the files in staging (to speed up commits - only format those that have changed) through to a linter/formatter (in our case Prettier).